packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -121,25 +121,43 @@ class Xcode {
     return _eulaSigned;
   }
 
+  final RegExp xcodeVersionRegex = new RegExp(r'Xcode ([0-9.]+)');
+  void _updateXcodeVersion() {
+    try {
+      _xcodeVersionText = processManager.runSync(<String>['/usr/bin/xcodebuild', '-version']).stdout.replaceAll('\n', ', ');
+      final Match match = xcodeVersionRegex.firstMatch(xcodeVersionText);
+      if (match == null)
+        return;
+
+      final String version = match.group(1);
+      final List<String> components = version.split('.');
+      _xcodeMajorVersion = int.parse(components[0]);
```

Comment:
That's what I get for picking this up on a Monday after writing most of it on a Friday! Done.